### ISSUE TITLE
undo for block deletion

### ISSUE THREAD

**Issue by najlitarvan** [tsunamayo/Starship-EVO#6171]
ability to press a button to undo the few last block removal actions while building, though the ability should only remember these blocks for only a few seconds and if there is a block placed in the area the block occupied, it should not be respawned

**Comment by Tackietacktack** [comment]
Yes, sometimes i wish its possible. And one step back would be enough if we delete a complicate CE. I think, One step is not to much for survival.

**Comment by najlitarvan** [comment]
@Tackietacktack ya, one block would be more then enough

**Comment by tsunamayo** [comment]
@Tackietacktack Hi, yeah but having it work with CE is quite complicated.
